packages/core/src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -90,13 +94,17 @@ export class Autocomplete extends React.PureComponent {
       itemToString,
       label,
       loading,
-      onChange
+      onChange,
+      onInputValueChange,
+      children,
+      ...autocompleteProps
```

Comment:
Since we're passing in `autocompleteProps` now, is there a reason we need to create a `const` for 
`itemToString`, `onInputValueChange`, and `onChange`?